spec/models/award_spec.rb

```diff
@@ -50,5 +50,6 @@
 
   context 'with associations' do
     it { should belong_to(:country) }
+    it { should have_many(:categories) }
```

Comment:
can you replace please should with is_expected, for all associations, thanks!
![image](https://user-images.githubusercontent.com/70888057/157506155-c7af4ca5-fb94-4d2b-8dfc-b4bdf0e05f89.png)

Reply:
https://github.com/thoughtbot/shoulda-matchers

Reply:
ok than!

Reply:
my bad